sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/NestedColumnAliasingSuite.scala

```diff
@@ -313,6 +313,25 @@ class NestedColumnAliasingSuite extends SchemaPruningTest {
     comparePlans(optimized, expected)
   }
 
+  test("Nested field pruning for Project and PosExplode") {
```

Comment:
Can you add JIRA ticket as usual? Thanks.

Reply:
Oh, BTW, this rule is not for NestedColumnAliasing actually. Can you move it to `ColumnPruningSuite`?

Reply:
Done.